pkg/shortcut/artifacts.go

```diff
@@ -3,109 +3,115 @@ package shortcut
+	"defenders":                          keys.DefendersWill,
+	"defender":                           keys.DefendersWill,
+	"desertpavilionchronicle":            keys.DesertPavilionChronicle,
+	"desertpavilion":                     keys.DesertPavilionChronicle,
+	"dpc":                                keys.DesertPavilionChronicle,
+	"vourukashasglow":                    keys.VourukashasGlow,
+	"echoesofanoffering":                 keys.EchoesOfAnOffering,
+	"echoes":                             keys.EchoesOfAnOffering,
+	"emblemofseveredfate":                keys.EmblemOfSeveredFate,
+	"emblem":                             keys.EmblemOfSeveredFate,
+	"eosf":                               keys.EmblemOfSeveredFate,
+	"esf":                                keys.EmblemOfSeveredFate,
+	"flowerofparadiselost":               keys.FlowerOfParadiseLost,
+	"paradiselost":                       keys.FlowerOfParadiseLost,
+	"fopl":                               keys.FlowerOfParadiseLost,
+	"gambler":                            keys.Gambler,
+	"gladiatorsfinale":                   keys.GladiatorsFinale,
+	"glad":                               keys.GladiatorsFinale,
+	"gladiators":                         keys.GladiatorsFinale,
+	"gildeddreams":                       keys.GildedDreams,
+	"gilded":                             keys.GildedDreams,
+	"gd":                                 keys.GildedDreams,
+	"goldentroupe":                       keys.GoldenTroupe,
+	"gt":                                 keys.GoldenTroupe,
+	"heartofdepth":                       keys.HeartOfDepth,
+	"hod":                                keys.HeartOfDepth,
+	"huskofopulentdreams":                keys.HuskOfOpulentDreams,
+	"husk":                               keys.HuskOfOpulentDreams,
+	"hood":                               keys.HuskOfOpulentDreams,
+	"instructor":                         keys.Instructor,
+	"ins":                                keys.Instructor,
+	"lavawalker":                         keys.Lavawalker,
+	"lw":                                 keys.Lavawalker,
+	"luckydog":                           keys.LuckyDog,
+	"maidenbeloved":                      keys.MaidenBeloved,
+	"maiden":                             keys.MaidenBeloved,
+	"mb":                                 keys.MaidenBeloved,
+	"marechausseehunter":                 keys.MarechausseeHunter,
+	"mh":                                 keys.MarechausseeHunter,
+	"martialartist":                      keys.MartialArtist,
+	"nighttimewhispersintheechoingwoods": keys.NighttimeWhispersInTheEchoingWoods,
+	"echoingwoods":                       keys.NighttimeWhispersInTheEchoingWoods,
+	"nightwoods":                         keys.NighttimeWhispersInTheEchoingWoods,
+	"nighttime":                          keys.NighttimeWhispersInTheEchoingWoods,
+	"nwitew":                             keys.NighttimeWhispersInTheEchoingWoods,
+	"nwew":                               keys.NighttimeWhispersInTheEchoingWoods,
```

Comment:
add:
- nighttimewhispers

remove:
- nightwoods
- nwitew
- nwew

**don't forget to adjust docs**